man/man8/zstreamdump.8

```diff
@@ -1,58 +0,0 @@
-'\" te
-.\" Copyright (c) 2009, Sun Microsystems, Inc. All Rights Reserved
-.\" The contents of this file are subject to the terms of the Common Development and Distribution License (the "License").  You may not use this file except in compliance with the License. You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE or http://www.opensolaris.org/os/licensing.
-.\"  See the License for the specific language governing permissions and limitations under the License. When distributing Covered Code, include this CDDL HEADER in each file and include the License file at usr/src/OPENSOLARIS.LICENSE.  If applicable, add the following below this CDDL HEADER, with
-.\" the fields enclosed by brackets "[]" replaced with your own identifying information: Portions Copyright [yyyy] [name of copyright owner]
-.TH ZSTREAMDUMP 8 "Aug 24, 2020" OpenZFS
-.SH NAME
-zstreamdump \- filter data in zfs send stream
-.SH SYNOPSIS
-.LP
-.nf
-\fBzstreamdump\fR [\fB-C\fR] [\fB-v\fR] [\fB-d\fR]
```

Comment:
Is there value to removing this manpage?  If so, do we need to be sure to mention it in the next release notes?

Reply:
The page was not removed.

Reply:
To be clear, this man page was replaced by a link to the `zstream` man page which now includes an additional line which states that `zstreamdump` is an alias for `zstream dump`.  I'm OK with consolidating the documentation to help make it easier to maintain.